dashboard/lib/tasks/asset_sync.rake

```diff
@@ -0,0 +1,11 @@
+namespace :assets do
+  desc 'Synchronize assets to S3'
+  task :sync => :environment do
+    # AWS CLI implements an optimized `sync` utility without any Ruby SDK equivalent.
+    `aws s3 sync #{CDO.dashboard_dir}/public/assets s3://#{CDO.assets_bucket}/#{rack_env}/assets --acl public-read --cache-control 'max-age=31536000'`
```

Comment:
I was going to ask about using --delete. Thanks for those metrics. If it's taken this long to accumulate 1.6GB, I don't think we ever have to worry about it.